alembic/versions/2024_03_18_24c1e9e1476d_remove_unused_sample_columns.py

```diff
@@ -0,0 +1,29 @@
+"""Remove unused sample columns
+
+Revision ID: 24c1e9e1476d
+Revises: b2fcb2ada306
+Create Date: 2024-03-18 14:59:13.832047
+
+"""
+
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import mysql
+
+# revision identifiers, used by Alembic.
+revision = "24c1e9e1476d"
+down_revision = "b2fcb2ada306"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.drop_column("sample", "sequence_start")
```

Comment:
<img width="185" alt="image" src="https://github.com/Clinical-Genomics/cg/assets/40887124/4905808a-387a-48fc-b4e8-9e97c844c14b">
We do have a few samples with this set

Reply:
Yes, I don't think it is significant though since they are so few and we don't use the field. I'm guessing these 5 were set manually.

What do you think?

Reply:
I would say it is fine, but on the other hand, I don't consider myself to have the final say in this.

Reply:
Given it is not used anywhere, it is just bloat that pollutes are code and data bases. I guess we just need to make sure that no one is manually setting and reading this field. I'll ask production

Reply:
I just saw now that this is a different column. This PR drops the `_is_external` column.
Apparently, status db has a `is_external` column which is not defined in our model.

Reply:
No ones uses these fields